src/Apps/Artwork/Components/Share.tsx

```diff
@@ -1,22 +1,34 @@
-import { Flex, Sans, Separator } from "@artsy/palette"
+import { color, Flex, Sans, Separator } from "@artsy/palette"
 import { IconName } from "Assets/Icons"
 import Icon from "Components/Icon"
 import React from "react"
 import styled from "styled-components"
 
 interface ShareProps {
   url: string
+  handleClose?: () => void
```

Comment:
Conventionally we use `handleClose` inside of a class for attaching to an action, and then the external "props" API is `onClose`.